lib/expectedConditions.ts

```diff
@@ -404,8 +403,8 @@ export class ProtractorExpectedConditions {
   }
 
   /**
-   * An expectation for checking that an element is either invisible or not
-   * present on the DOM. This is the opposite of 'visibilityOf'.
+   * An expectation for checking that an element is invisible on the DOM.
```

Comment:
I initially opened this PR exactly to change this piece of text. The minor formatting fix above was just some "gardening".

You are quite right, on closer examination of the source code, the intent of `invisibilityOf` is as you say.

However, I have since opened this issue https://github.com/angular/protractor/issues/3777 because invisibilityOf seems to have a race condition, which is why it cannot be used in its current state for waiting for elements to be removed from the DOM.